src/services/UserInfoRequestProcessor.ts

```diff
@@ -32,20 +34,16 @@ export class UserInfoRequestProcessor {
     private readonly verifiableCredentialService: VerifiableCredentialService;
 
     constructor(logger: Logger, metrics: Metrics) {
-    	if (!SESSION_TABLE) {
-    		logger.error("Environment variable SESSION_TABLE is not configured");
-    		throw new AppError("Service incorrectly configured", HttpCodesEnum.SERVER_ERROR );
-    	}
-    	if (!KMS_KEY_ARN) {
-    		logger.error("Environment variable KMS_KEY_ARN is not configured");
+    	if (!SESSION_TABLE || !ISSUER || !KMS_KEY_ARN) {
```

Comment:
Could the same change be made to AuthProcessor pls to keep them all consistent

Reply:
Done.